core/common/src/BriefcaseTypes.ts

```diff
@@ -25,10 +25,16 @@ export enum DownloadBriefcaseStatus {
   Error,
 }
 
-/** Operations allowed when synchronizing changes between the Briefcase and iModelHub
+/** Whether a briefcase is editable or may only accept incoming changesets from iModelHub
  * @public
  */
-export enum SyncMode { FixedVersion = 1, PullAndPush = 2, PullOnly = 3 }
+export enum SyncMode {
+  FixedVersion = 1,
+  /** create a briefcase that can be edited. A unique briefcaseId is assigned by iModelHub */
+  PullAndPush = 2,
+  /** use [BriefcaseIdValue.Standalone](%backend). This makes a briefcase that can accept changesets from iModelHub but can never create changesets. */
+  PullOnly = 3

```

Comment:
need trailing `,` for linter

Reply:
OK, but `npm run lint` doesn't flag this on my box. Do I have something wrong?

Reply:
Strange, maybe we changed that lint rule to not require this anymore...